scripts/cnn_variant_wdl/cram2filtered.json

```diff
@@ -0,0 +1,21 @@
+{
+	"Cram2FilteredVcf.input_cram": "gs://broad-dsde-methods/palantir/ClinicalGenomeTruthSets/NA12878_NA12878_O2D1_SM-G947X_v1.cram",
+	"Cram2FilteredVcf.reference_fasta": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta",
+	"Cram2FilteredVcf.reference_dict": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
+	"Cram2FilteredVcf.reference_fasta_index": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai",
+	"Cram2FilteredVcf.output_prefix": "g947x_o2d1v1_na12878",
+	"Cram2FilteredVcf.architecture_json":"gs://broad-dsde-methods/cnn-variant-score/architectures/small_2d_tf_model.json",	
+	"Cram2FilteredVcf.architecture_hd5":"gs://broad-dsde-methods/cnn-variant-score/architectures/small_2d_tf_model.hd5",
+	"Cram2FilteredVcf.tensor_type":"read_tensor",
+	"Cram2FilteredVcf.gatk_override": "gs://broad-dsde-methods/sam/jars/gatk.jar",
+	"Cram2FilteredVcf.picard_jar": "gs://broad-dsde-methods/skwalker/picard_2.12.1.jar",
+	"Cram2FilteredVcf.calling_intervals": "gs://broad-references/hg38/v0/wgs_calling_regions.hg38.interval_list",
+	"Cram2FilteredVcf.gatk_docker": "samfriedman/p3",
+	"Cram2FilteredVcf.preemptible_attempts": 0,
+	"Cram2FilteredVcf.inference_batch_size": 1,
+	"Cram2FilteredVcf.transfer_batch_size": 1,
+	"Cram2FilteredVcf.mem_gb": 7,
+	"Cram2FilteredVcf.extra_args": "--dont-trim-active-regions -stand-call-conf 0 -A Coverage -A ChromosomeCounts -A BaseQuality -A FragmentLength -A MappingQuality -A ReadPosition ",
```

Comment:
I have some concerns about these args.  Do you need all those annotations? And why the stand-call-conf 0?  That's going to produce a lot of extra variants that won't be in the genotyped VCF. Also I just remembered that active regions overlap periodically -- is that a problem for your model?

Reply:
stand-call-conf at 0 was to try to squeeze more negative examples out for training, but for inference we can ditch this.  I do see slightly better performance by including the extra annotations especially on HG38.  I discard the artificial haplotypes so I think active region overlap shouldn't be an issue.

Reply:
I beg to differ re:overlap.  There will be reads at either end of the artificial haplotypes.  This is an issue Ben had to deal with for the ExAC/gnomAD browser.  Admittedly that was GATK3, but we haven't changed the active region detection or the bamout (substantially).  I'm trying to dig up a screenshot, but slack is sucking again.  Could you do a quick check?  (A super slow pysam solution would be fine.)  I'm not saying it's a fatal issue, but it might mean that variants within a certain distance other variants have an additional signal.

I also have a note from GATK3: "uninformative reads are set to zero mapping quality to enhance visualization"  You don't have an overwhelming number of MQ0s, right?

Reply:
I don't see many MQ0s in the bamouts this generates.  I'm confused about the overlapping active regions.  Will reads be excluded from an active region if they overlap another?